sdk/eventhub/azure-eventhub-checkpointstoreblob/setup.py

```diff
@@ -67,7 +67,9 @@
     zip_safe=False,
     packages=find_packages(exclude=exclude_packages),
     install_requires=[
-        'azure-storage-blob<13.0.0,>=12.0.0',
+        "azure-core<2.0.0,>=1.2.2",
```

Comment:
I would clarify with a comment at least that it's coming from storage SDK for clarity. Today we remember, but in 6 months if the project moved along the team?

Reply:
Added comments for azure-core, msrest and cryptography